download.inc

```diff
@@ -303,3 +303,51 @@ function dwsim_flowsheet_download_lab()
 		drupal_goto('lab-migration/lab-migration-run');
 	}
 }
+
+function dwsim_flowsheet_download_proposals()
+{
+	$root_path = dwsim_flowsheet_path();
+	$result = db_query("SELECT e.contributor_name as Contirbutor_Name, u.mail as Email_ID, e.project_title as Title, e.contact_no as Contact, from_unixtime(actual_completion_date,'%Y') as Year FROM dwsim_flowsheet_proposal as e JOIN users as u ON e.uid = u.uid WHERE e.approval_status=3 ORDER BY actual_completion_date DESC");
+//var_dump($result->rowCount());die();
+	 //$all_proposals_q = $result->execute();
+	 $participants_proposal_id_file = $root_path . "participants-proposals.csv";
+	 $fp = fopen($participants_proposal_id_file, "w");
+            /* making the first row */
+            $items = array(
+                'Contirbutor_Name',
+                 'Email_ID',
+                 'Title',
+                 'Contact', 
+                 'Year'
```

Comment:
Add columns for creation date, university, date of approval, and status of the proposal

Reply:
Rename the column name as Date of Completion, do not use underscore in the column name